src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class RightTriangle extends FigureColor {
```

Comment:
The class name 'RightTriangle' should not extend 'FigureColor'. It seems like 'FigureColor' is being used to set behavior, which is not recommended. Consider using an interface to define behavior and separate classes or a composition approach to handle coloring.